routeros_api/api.py

```diff
@@ -9,20 +9,20 @@
 from routeros_api import resource
 
 
-def connect(host, username='admin', password='', port=8728, force_old_login=False):
-    return RouterOsApiPool(host, username, password, port, force_old_login).get_api()
+def connect(host, username='admin', password='', port=8728, insecure_login=False):
+    return RouterOsApiPool(host, username, password, port, insecure_login).get_api()
 
 
 class RouterOsApiPool(object):
     socket_timeout = 15.
 
-    def __init__(self, host, username='admin', password='', port=8728, force_old_login=False):
+    def __init__(self, host, username='admin', password='', port=8728, insecure_login=False):
```

Comment:
I don't like the name of this parameter. In future people will have to by default set it to True and it will make them think that they are doing something wrong. Maybe "plaintext_login"?

MD5 is not that secure unless you are using a long password. If you have the challenge and the final md5 you can brute force it. I just tested it on my laptop - 8 character password was cracked in 10 min and on better GPU it could run 40x faster, so for the md5 to be secure enough you have to have at least 13 characters in your password (and it should be complicated one).
To try it yourself:
```
hashcat -m 10 90be209e598d48fbd6a5e144a7a9ebf0:ebddd18303a54111e2dea05a92ab46b4 --hex-charset --hex-salt -a 3 --increment '00?l?l?l?l?l?l?l?l'
```

Let's keep it "secure" by default for now, but that should be changed in the future when people will start using new RouterOs version.